packages/plugin-lighthouse/src/lib/utils.ts

```diff
@@ -68,3 +75,89 @@ export function validateOnlyAudits(
   }
   return true;
 }
+
+type UnsupportedDetail =
+  | Details.CriticalRequestChain
+  | Details.List
+  | Details.TreemapData
+  | Details.Screenshot
+  | Details.Filmstrip
+  | Details.DebugData;
+type UnsupportedDetailTypes = UnsupportedDetail['type'];
+
+export function toAuditOutputs(
+  lhrAudits: Result[],
+): AuditOutputs {
+  return Object.values(lhrAudits).map(
+    ({
+      id: slug,
+      score,
+      numericValue: value = 0, // not every audit has a numericValue
+      details,
+      displayValue,
+    }: Result) => {
+      const auditOutput: AuditOutput = {
+        slug,
+        score: score ?? 1, // score can be null
+        value,
+        displayValue,
+      };
```

Comment:
Interesting scenario. Not sure if `score: 1, value: 0` is the best way to represent such audits, filtering them out could also be an option :thinking: Which Lighthouse audits does this concern?

Reply:
There are multiple scenarios that a was able to track down:
- The provided URL is not accessible. In this case all audits have a score of `null` but are still present in the result.
- If the URL is valid, the following `scoreDisplayMode` have a score of `null`:
  -  `informative`
  - `notApplicable`
  - `manual`
  
  Here a screenshot:
  
<img width="1099" alt="Screenshot 2024-02-28 at 16 16 32" src="https://github.com/code-pushup/cli/assets/10064416/d0dc107d-d607-4686-bc3c-d29f773c6fcb">

Reply:
(I'm fine with this being a separate issue, BTW :top:)

Reply:
Removed all code+tests related to details and opened #530